arangod/RestServer/UpgradeFeature.cpp

```diff
@@ -148,6 +149,10 @@ void UpgradeFeature::validateOptions(std::shared_ptr<ProgramOptions> options) {
       server().getFeature<ReplicationFeature>();
   replicationFeature.disableReplicationApplier();
 
+  cluster::FailureOracleFeature& failureOracleFeature =
```

Comment:
Can you please move this into the list of `_nonServerFeatures` that is handed in from the outside.This way we have it at least in the same position as the other force-disabled features.

Reply:
I would actually prefer to not have this, but only have the "onlyEnabledWith<>".

Reply:
Reverted this file back to devel version